File: framework/src/main/java/com/software/dm/swallow/stormy/algoac/pojo/VagueResultEntity.java
package com.software.dm.swallow.stormy.algoac.pojo;

/**
 * @author DM
 * @version v1.0.0.1
 * @Description
 * @date 2017
 */
public final class VagueResultEntity {

    /**
     * @param obj
     * @param mustEqual
     */
    public VagueResultEntity(Object obj, boolean mustEqual) {
        this.obj = obj;
        this.mustEqual = mustEqual;
    }

    public VagueResultEntity() {

    }

    // obj
    private Object obj;
    // string is endding
    private boolean mustEqual = false;

    public Object getObj() {
        return obj;
    }

    public void setObj(Object obj) {
        this.obj = obj;
    }

    public boolean isMustEqual() {
        return mustEqual;
    }

    public void setMustEqual(boolean mustEqual) {
        this.mustEqual = mustEqual;
    }

    @Override
    public String toString() {
        return "VagueResultEntity [obj=" + obj + ", mustEqual=" + mustEqual + "]";
    }

}

File: framework/src/main/scala/com/software/dm/swallow/stormy/scala/algoac/pojo/VagueResultEntity.scala
package com.software.dm.swallow.stormy.scala
package algoac
package pojo

/**
  * @author darlingming@126.com
  * @param obj
  * @param mustEqual
  * 2017
  * @version v1.0.0.1
  */
final class VagueResultEntity(var obj: Any, var mustEqual: Boolean) extends  Serializable{


  def getObj: Any = obj

  def setObj(obj: Any): Unit = {
    this.obj = obj
  }

  def isMustEqual: Boolean = mustEqual

  def setMustEqual(mustEqual: Boolean): Unit = {
    this.mustEqual = mustEqual
  }

  override def toString: String = "VagueResultEntity [obj=" + obj + ", mustEqual=" + mustEqual + "]"
}

